packages/playwright-ct-react/registerSource.mjs

```diff
@@ -40,7 +40,13 @@ function __pwRender(value) {
     if (isJsxComponent(v)) {
       const component = v;
       const props = component.props ? __pwRender(component.props) : {};
-      return { result: __pwReact.createElement(/** @type { any } */ (component.type), { ...props, children: undefined }, props.children) };
+      const {children, ...propsWithoutChildren} = props;
```

Comment:
Nice catch @JReinhold! I'm looking forward to explore the upcomming Storybook/Playwright CT integration!

The change looks good from a functional point of view, particularly since it mirrors the workings of React. I only wonder if we should dedupe this code by moving it into `@playwright/experimental-ct-core`?

Reply:
Thanks!

That's up to you. I personally wouldn't recommend it, I think this quirk is very specific to React and so you'd have React-specific code in core. It's duplicated between React and React17, and if that alone warrants deduplication then I'd think the whole file would be in core because it's duplicated.

But if you want me to move it I can do that.

Reply:
Looks good. I'll follow up with the lint - turns out out eslint does not apply to mjs